modules/ROOT/pages/graph-analytics/index.adoc

```diff
@@ -1,13 +1,16 @@
 [[graph-analytics-aura]]
-= Graph analytics on Aura
-:description: This page describes how to run graph analytics on Aura.
+= Graph analytics in Aura
+:description: This page describes how to run graph analytics in Aura.
 :page-aliases: aurads/index.adoc
 
-There are two ways to run graph analytics on Aura:
+If your data is in Aura or you plan to move it to Aura, you have several options to run graph analytics:
 
-* Create an AuraDB Professional instance with the <<aura-gds-plugin>>, or enable the plugin on an existing AuraDB Professional instance.
+* Enable <<aura-gds-plugin,Graph Analytics>> on an existing AuraDB instance (Professional only).
+* Use <<aura-gds-serverless,Graph Analytics Serverless>> from an existing AuraDB instance (Business Critical or Virtual Dedicated Cloud).
```

Comment:
```suggestion
* Enable <<aura-gds-plugin,Graph Analytics>> on an existing AuraDB instance (Professional).
* Use <<aura-gds-serverless,Graph Analytics Serverless>> from an existing AuraDB instance (Business Critical, Virtual Dedicated Cloud).
```
I reacted to the use of 'only' in the one case. this is a weak suggestion, but I'll make it anyway